README.md: Product description, features, installation

joswinter opened the pull request:
Added product description, features and installation to the README.md.
Should fix #89 Improve the readme

joswinter committed:
README.md: Product description, features, installation

Added product description, features and installation to the README.md

jorenham commented on README.md:
```diff
@@ -1,2 +1,29 @@
+
+
+- View a single or multiple DNA strands as a graph to research the structure. In a unified form where every sequence segment has the same coordinate system and empty sequence segments are also visualized.
+- Create a phylogenetic tree based on several DNA strands so that the user can compare them and observe the timeline and the common ancestors of several functionalities.
```
`Create` -> `Display`

jorenham commented on README.md:
```diff
@@ -1,2 +1,29 @@
+
+The application can be executed by downloading or cloning the source. 
+
+To start researching your DNA sequence graph you need your files in the graph format which consists of two files with `.node.graph` and `.edge.graph` as extension.
```
The `.node.graph` and `.edge.graph` files also need to have the same filename

joswinter committed:
Implemented Joren's comments

- Display -> Create changed
- Identical filenames + example added
- Context Project reference added
